web/business-registry-dashboard/app/components/Table/AffiliatedEntity/StatusDetails.vue

```diff
@@ -76,7 +76,15 @@ const generateMessage = (status: string | { type: string, data: any }): Message
       type: status.type
     }
   }
-
+  // EXPIRING in 14 Days alert - includes the type of entity that's expiring soon
+  if (status.type === EntityAlertTypes.EXPIRING_SOON) {
+    return {
+      message: t(`entityAlertTypes.${EntityAlertTypes.EXPIRING_SOON}`, status.data),
+      colour: 'text-bcGovColor-caution',
+      priority: 3.5, // Medium priority - higher than BADSTANDING (priority 4)
```

Comment:
First, I think we don't need to worry about entity alerts because this filing is still a draft.

Second, let's stick with integral numbers. (I think of this like run levels or task priorities.) Also, it's OK to have 2 alerts at the same priority. In my opinion, this can be the same priority as Expired (because it can never be both).

Reply:
Changed, thanks.